examples/3l-node/cli.rs

```diff
@@ -625,12 +625,39 @@ fn list_payments(node: &LightningNode) -> Result<(), String> {
         println!("      Preimage:     {:?}", payment.preimage);
         println!("      Description:  {}", payment.description);
         println!("      Invoice:      {}", payment.invoice_details.invoice);
-        println!();
+        println!("      Offer:        {}", offer_to_string(payment.offer));
     }
 
     Ok(())
 }
 
+fn offer_to_string(offer: Option<OfferKind>) -> String {
+    match offer {
+        Some(OfferKind::Pocket {
```

Comment:
Probably the right way would be to implement the `Display` trait for `OfferKind` and handle the Option here separately

Reply:
Well, `Display` could be implemented within `3L` where `OfferKind` itself is implemented, couldn't it?